src/main/java/org/spongepowered/common/mixin/core/command/MixinCommandBlockSource.java

```diff
@@ -36,7 +36,8 @@
 import org.spongepowered.common.interfaces.IMixinSubject;
 
 @NonnullByDefault
-@Mixin(value = {TileEntityCommandBlock.class, EntityMinecartCommandBlock.class}, targets = IMixinCommandSender.SIGN_CLICK_SENDER)
+@Mixin(value = {TileEntityCommandBlock.class, EntityMinecartCommandBlock.class},
+       targets = {IMixinCommandSender.SIGN_CLICK_SENDER_1, IMixinCommandSender.SIGN_CLICK_SENDER_2})
```

Comment:
`MixinCommandBlockSource` is a `CommandBlockSource` which 
```
Represents a CommandBlock source, either a placed block or a CommandBlockMinecart.
```


They should not be targeting IMixinCommandSender#SIGN_CLICK_SENDER_1 (and 2)